plugins/ipam/ipstore/ipstore.go

```diff
@@ -159,6 +173,53 @@ func (manager *IPManager) Release(ip string) error {
 	return nil
 }
 
+// ReleaseByID release the key-value pair by id
+func (manager *IPManager) ReleaseByID(id string) (string, error) {
+	// TODO improve this part by implement listing all the kv pairs
+	// libkv library only provide list all key-value pairs with prefix of key, and
+	// if the result is empty it will return store.ErrKeyNotFound error
+	kvPairs, err := manager.client.List(IPPrefix)
+	if err == store.ErrKeyNotFound {
+		return "", errors.Errorf("release ipstore: no ip associated with the id: %s", id)
```

Comment:
In this case `store.ErrKeyNotFound` isn't really an error, it's just because today we don't have a better way to get all the key-value pairs from the db. If we can fix #37 then we will get rid of this error.